test/__snapshots__/plugin.test.ts.snap

```diff
@@ -143,3 +143,17 @@ exports[`default options 1`] = `
     </body>
   </html>"
 `;
+
+exports[`mini-css-extract-plugin 1`] = `
```

Comment:
I don't see anything specific to the CSS plugin here, do we need this snapshot?

Reply:
Yeah, I updated the test to include some CSS.